cmake/Modules/FindDotNETExtra.cmake

```diff
@@ -58,6 +58,50 @@ function(add_dotnet_executable _TARGET_NAME)
   )
 endfunction()
 
+function(add_dotnet_test _TARGET_NAME)
+  cmake_parse_arguments(_add_dotnet_test
+    ""
+    ""
+    "SOURCES;INCLUDE_DLLS;INCLUDE_NUPKGS;INCLUDE_REFERENCES"
+    ${ARGN}
+  )
+
+  set(CSHARP_TARGET_FRAMEWORK "netcoreapp2.0")
```

Comment:
CSHARP_TARGET_FRAMEWORK is set by the file `FindCSBuild.cmake` -- were you explicit here for unit tests on purpose, or should the same dotnet target framework be used as what the rest of the project was built with?

Reply:
Using netstandard1.1 with xUnit, I got a `"CS5001 error: Program does not contain a static 'Main' method suitable for an entry point"`. I couldn't resolve this problem, as every example that I found used netcoreapp2.0. 

CSHARP_TARGET_FRAMEWORK is set by default to "netstandard1.1 in the file FindCSBuild.cmake:34. I have changed to "netcoreapp2.0" for being used as default in rcldotnet at everything looks working ok. I only tested in Linux. 

In [this](https://docs.microsoft.com/en-us/archive/msdn-magazine/2016/april/net-core-net-goes-cross-platform-with-net-core) article says that UWP apps can only reference the .NET standard class library template, not .NET Core class library template, so using "netcoreapp2.0" as default is not an option by default.

Summarizing: We need "netcoreapp2.0" for testing with XUnit and "netstandard1.1" for the rest of the project if we want to run UWP.

Reply:
Makes sense -- thanks for taking the time to help me understand.